lib/active_admin/reform/dsl.rb

```diff
@@ -0,0 +1,28 @@
+require 'active_support/concern'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.